core/food_apis/update_manager.py

```
@@ -816,8 +820,11 @@ async def _load_backup(self, source: str, version: str) -> Dict[str, UnifiedFood
                 if not isinstance(food_data, dict) or not required.issubset(food_data.keys()):
                     continue
                 foods[name] = UnifiedFoodItem(**food_data)
-            except Exception:
+            except Exception as e:  # nosec B112
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)